apps/graph/graph/tangent_graph_controller.cpp

```diff
@@ -100,4 +100,31 @@ bool TangentGraphController::handleEnter() {
   return true;
 }
 
+bool TangentGraphController::handleEvent(Ion::Events::Event event) {
+  if (event == Ion::Events::Up) {
+    m_recordDelegate->moveUp();
+    m_graphView->selectRecord(m_recordDelegate->getRecord());
+    // TODO maybe do: Very ugly workaround... Needs something better to reload the data banner
```

Comment:
Do you try with the AppsContainer reload function ?

Reply:
Yup and it's not working (just tried xd).